executor/executor_test.go

```diff
@@ -43,6 +43,23 @@ func (s *testSuite) TearDownSuite(c *C) {
 	s.store.Close()
 }
 
+func (s *testSuite) TestAdmin(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists admin_test")
+	tk.MustExec("create table admin_test (id int PRIMARY KEY AUTO_INCREMENT, c1 int, c2 int, c3 int default 1)")
+	tk.MustExec("insert admin_test (c1) values (1),(2),(NULL)")
+	r, err := tk.Exec("admin show ddl")
+	c.Assert(err, IsNil)
+	row, err := r.Next()
+	c.Assert(err, IsNil)
+	c.Assert(row, NotNil)
+
+	r, err = tk.Exec("admin check table admin_test")
+	c.Assert(err, IsNil)
+	c.Assert(r, IsNil)
+}
```

Comment:
The test didn't cover the case when check table returns error which is the main purpose.